tests/lib/rules/no-force-true.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-force-true')
+
+const RuleTester = require('eslint').RuleTester
+
+const errors = [{ messageId: 'unexpected' }]
+const parserOptions = { ecmaVersion: 6 }
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+let ruleTester = new RuleTester()
+
+ruleTester.run('no-force-true', rule, {
+
+  valid: [
+    { code: 'cy.get(\'button\').click()', parserOptions },
+    { code: 'cy.get(\'button\').click({multiple: true})', parserOptions },
+    { code: 'cy.get(\'button\').dblclick()', parserOptions },
+    { code: 'cy.get(\'input\').type(\'somth\')', parserOptions },
+  ],
+
+  invalid: [
+    { code: 'cy.get(\'button\').click({force: true})', parserOptions, errors },
```

Comment:
could you add a test for `cy.get('div').find('.foo').click({ force: true })` ?

Reply:
yep